java/client/src/org/openqa/selenium/print/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("//java:defs.bzl", "java_library")
+
+java_library(
+    name = "print",
+    srcs = glob(["**/*.java"]),
+    visibility = [
+        "//visibility:public",
```

Comment:
This should only be visible to the `java_export` target that this is part of (and, if necessary, other libraries that are part of that target). Public visibility is almost always a mistake for something that isn't meant to ship as a standalone component.